frontend/src/app/modules/interventions/list/table/table.component.scss

```diff
@@ -1,16 +1,86 @@
 table {
-    width: 100%;
+  width: 100%;
+  table-layout: fixed;
 
-    .mat-row:hover {
-        cursor: pointer;
-        background: #ccc5;
-    }
+  .mat-row:hover {
+    cursor: pointer;
+    background: #ccc5;
+  }
 }
 
 .mat-column-description {
+  line-height: 24px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+@media screen and (min-width: 701px) {
```

Comment:
Kind of 'same' with scss variable. Is there a way in scss to add one pixel to an existing variable? :)

Reply:
Yeah, you can `max-width: $break-mobile - 1` fixed 👍